src/lib/pubkey/pk_ops.h

```diff
@@ -33,7 +38,7 @@ namespace PK_Ops {
 /**
 * Public key encryption interface
 */
-class Encryption {
+class BOTAN_PUBLIC_API(3, 3) Encryption {
```

Comment:
These should be `BOTAN_UNSTABLE_API`.

Using this interface requires deriving, which is not covered by our current SemVer guarantee (`doc/sem_ver.rst`) and already I know we have to change at least the decryption API to support some side channel mitigations.

Reply:
Done.